### PR TITLE
[TASK] Drop definition of PATH_site

### PR DESCRIPTION
**Pull request by lolli42**

### CODE REVIEW

File: Resources/Core/Build/UnitTestsBootstrap.php [hunk 1]
```diff
@@ -30,15 +30,34 @@
...
-    $testbase->createDirectory(PATH_site . 'uploads');
+
+    $requestType = \TYPO3\CMS\Core\Core\SystemEnvironmentBuilder::REQUESTTYPE_BE | \TYPO3\CMS\Core\Core\SystemEnvironmentBuilder::REQUESTTYPE_CLI;
+    \TYPO3\CMS\Core\Core\SystemEnvironmentBuilder::run(0, $requestType);
+
+    $testbase->createDirectory(\TYPO3\CMS\Core\Core\Environment::getPublicPath() . '/typo3conf/ext');
+    $testbase->createDirectory(\TYPO3\CMS\Core\Core\Environment::getPublicPath() . '/typo3temp/assets');
+    $testbase->createDirectory(\TYPO3\CMS\Core\Core\Environment::getPublicPath() . '/typo3temp/var/tests');
+    $testbase->createDirectory(\TYPO3\CMS\Core\Core\Environment::getPublicPath() . '/typo3temp/var/transient');
+    $testbase->createDirectory(\TYPO3\CMS\Core\Core\Environment::getPublicPath() . '/uploads');
```
Comment (bmack) [inline Resources/Core/Build/UnitTestsBootstrap.php]: you can get rid of this line btw.

File: Resources/Core/Build/UnitTestsBootstrap.php [hunk 2]
```diff
@@ -30,15 +30,34 @@
...
-    $testbase->createDirectory(PATH_site . 'typo3conf/ext');
-    $testbase->createDirectory(PATH_site . 'typo3temp/assets');
-    $testbase->createDirectory(PATH_site . 'typo3temp/var/tests');
-    $testbase->createDirectory(PATH_site . 'typo3temp/var/transient');
-    $testbase->createDirectory(PATH_site . 'uploads');
+
+    $requestType = \TYPO3\CMS\Core\Core\SystemEnvironmentBuilder::REQUESTTYPE_BE | \TYPO3\CMS\Core\Core\SystemEnvironmentBuilder::REQUESTTYPE_CLI;
+    \TYPO3\CMS\Core\Core\SystemEnvironmentBuilder::run(0, $requestType);
+
+    $testbase->createDirectory(\TYPO3\CMS\Core\Core\Environment::getPublicPath() . '/typo3conf/ext');
```
Comment (bmack) [inline Resources/Core/Build/UnitTestsBootstrap.php]: could you use the respective usages within the environment API class instead?

Comment (bmack) [reply Resources/Core/Build/UnitTestsBootstrap.php]: also below....

Comment (lolli42) [reply Resources/Core/Build/UnitTestsBootstrap.php]: > could you use the respective usages within the environment API class instead?

will do with a next patch later. not sure if varPath is always correctly used / initialized in tests, yet.

File: Resources/Core/Build/UnitTestsBootstrap.php (hunk 1 above)
Comment (lolli42) [reply Resources/Core/Build/UnitTestsBootstrap.php]: leads to a single test fail. let's do that separately

**Pull request merged**